pagerduty/resource_pagerduty_service.go

```diff
@@ -276,7 +276,6 @@ func resourcePagerDutyServiceRead(d *schema.ResourceData, meta interface{}) erro
 	}
 
 	d.Set("name", service.Name)
-	d.Set("status", service.Status)
```

Comment:
Removing this effectively renders the `status` field useless.
This should be kept to be able to read the current status of service.

I'd rather alter `buildServiceStruct()` to not set `pagerduty.Service.status` as it is a `Computed` field:
 https://github.com/terraform-providers/terraform-provider-pagerduty/blob/9aca2da0cae321fd15db99d82bcaa6057e771e71/pagerduty/resource_pagerduty_service.go#L192

Reply:
Patrick,
the status for this matter is useless anyway, but I see your point and I agree that is better to remove it from buildServiceStruct, updated the PR